### DISCUSSION TITLE
Unable to push changes on QField Cloud

### DISCUSSION THREAD

**Discussion by olbapablo** [opengisch/QField#2995] [Q&A]
I am trying to push the changes of several tablets to Qfield Cloud on a project. The problem is that some of the tablets can synchronize well but others always recieve the message that Qfield is unable to push changes. 

I have numerous photographs in the project so I suspect it might be the problem, but I cant find any documentation about the limits of the cloud in size and/or number of files.

**Comment by suricactus** [comment]
Can you check the QField logs and share them with us?

**Reply by olbapablo** [reply]
Yes the log messages are like this: 

_Failed to upload delta file, reason:
Error transferring https://app.qfield.cloud/api/v1/deltas/c2b950d4-6609-4636-a07e-35bcc82992af/ - server replied: Internal Server Error
[QF/] 
[QF/] 
[HTTP/500] https://app.qfield.cloud/api/v1/deltas/c2b950d4-6609-4636-a07e-35bcc82992af/ Server Error.{"code":["unknown_error"],"message":["QFieldcloud Unknown Error"]}
{"code":["unknown_error"],"message":["QFieldcloud Unknown Error"]}
Error transferring https://app.qfield.cloud/api/v1/deltas/c2b950d4-6609-4636-a07e-35bcc82992af/ - server replied: Internal Server Error
Attachment file '/data/data/ch.opengis.qfield/files/DCIM/street-picture_20220614144549907.jpg' does not exist_

**Reply by ntgiac95** [reply]
How to solve this  problem, my qfield on android had a problem like this
